app/controllers/v1/leaders_controller.rb

```diff
@@ -45,5 +47,40 @@ def welcome_letter_for_leader(leader)
         address: leader.address
       )
     end
+
+    def welcome_message(leader)
+      token = access_token leader
+      return if token.nil?
```

Comment:
That's what we're doing elsewhere.

https://github.com/hackclub/api/blob/master/app/models/leader.rb#L84